package.json

```diff
@@ -11,6 +11,7 @@
     "start": "next start"
   },
   "dependencies": {
+    "isomorphic-unfetch": "^3.0.0",
```

Comment:
So I tried to use yarn but it gave me an error message of some kind. Likely because I wasn't using the proper syntax, but I couldn't figure it out ... so I just tried NPM, didn't know it was going to create a new file or anything. I'll try and do what you are saying, but may need some help.